.changeset/blue-colts-film.md

```diff
@@ -0,0 +1,23 @@
+---
+'@astrojs/db': minor
+---
+
+Remove `AstroDbIntegration` type
+
+Astro integration hooks can now be extended and as such `@astrojs/db` no longer needs to declare it's own integration type. Using `AstroIntegration` will have the same type.
+
+If you were using the `AstroDbIntegration` type, apply this change to your integration code:
+
+```diff
+- import { defineDbIntegration, type AstroDbIntegration } from '@astrojs/db/utils';
++ import { defineDbIntegration } from '@astrojs/db/utils';
```

Comment:
I left it for backwards compatibility. No migration steps are needed for anyone who didn't look under the hood and just followed the docs.

Since this is for integration authors making integrations for a still beta platform, I'm fine with removing it completely (those are power users that should be fine with a small migration)

Reply:
Got it, I'll ask core about it

Reply:
Let's keep it for now, removal can happen in a later PR